release-tools/debian-package-builder

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/make -f
+# This program allows one to, relatively easily, iterate on Debian packaging in
+# a way that is extremely close to that occurring in the real Debian archive.
+#
+# The key aspect is that packaging is done on a machine running Debian (either
+# 9 or 10), with the real Debian kernel, with real build-packages (that is,
+# devoid of any patches potentially present only in Ubuntu), with correct
+# pristine build environment, unspoiled by convenience packages often present on
+# workstation environments. 
+#
+# The system is "portable", being able to execute on anything capable of
+# running Multipass and apt-cacher-ng. The workflow is as follows:
+#
+# - prepare host environment:
+#   $ snap install multipass --classic --edge
+#   $ apt install apt-cacher-ng
```

Comment:
It'd be nice to have a switch/option to not use apt-cacher-ng if it's not in the host distro repositories (as it isn't in Arch). Not a blocker though.

Reply:
Without this iteration is extremely painful. I may explore a snap option.